spec/features/workflow/clear_scheduled_publishing_datetime_spec.rb

```diff
@@ -4,57 +4,47 @@
   scenario do
     given_there_is_an_edition_with_a_set_scheduled_publishing_datetime
     when_i_visit_the_summary_page
-    and_i_click_on_publish_date
+    and_i_click_change_date
     then_i_see_the_existing_scheduled_publishing_date_and_time
     when_i_click_clear_date
-    then_i_see_the_default_scheduled_publishing_date_and_time
+    then_the_content_no_longer_has_a_scheduled_publishing_date_and_time
   end
 
   def given_there_is_an_edition_with_a_set_scheduled_publishing_datetime
     @datetime = Time.zone.now.advance(days: 2).midday
+    @scheduled_date = @datetime.strftime("%-d %B %Y")
     @edition = create(:edition, scheduled_publishing_datetime: @datetime)
   end
 
   def when_i_visit_the_summary_page
     visit document_path(@edition.document)
+    expect(page).to have_content("Proposed to publish at 12:00pm on #{@scheduled_date}")
```

Comment:
Could you help me understand why this line is here? I think it would be confusing for the tests to fail   on this line, as it's a `when` method.

Reply:
It's to make it clear that it's present at that point, and we are then expecting it not to be present in line 47.